### ISSUE TITLE
Can you please share the complete training scripts ?

### ISSUE THREAD

**Issue by yugeshav** [YangangCao/TRU-Net#1]
It will be good for the researchers / students if you share the working version of training script

**Comment by YangangCao** [comment]
to be finished...

**Comment by yugeshav** [comment]
ok, any timeline you are thinking ??

**Comment by amirpashamobinitehrani** [comment]
following @henryxrl 's comment, there is seems to be a Phase-aware β-sigmoid mask, attempting to separate the source, reverberated and noise parts of signal. I can't seem to find that in the implementation. Do you think that is part of the training step?

**Comment by maxwellmckinnon** [comment]
The phase aware part of the paper is not implemented.
